build.gradle

```diff
@@ -28,7 +28,7 @@ dependencies {
 	mappings "net.fabricmc:yarn:${project.yarn_mappings}:v2"
 	modImplementation "net.fabricmc:fabric-loader:${project.loader_version}"
 
-    include(implementation(annotationProcessor("io.github.llamalad7:mixinextras-fabric:0.2.0")))
+    //include(implementation(annotationProcessor("io.github.llamalad7:mixinextras-fabric:0.2.0")))
```

Comment:
Remove this line entirely.

Reply:
resolved in [07191ee](https://github.com/RelativityMC/raise-sound-limit-simplified/pull/7/commits/07191eed6c24466de1475e2674cd9080a78e0670)